src/birdtrayapp.cpp

```diff
@@ -68,6 +68,26 @@ BirdtrayApp::BirdtrayApp(int &argc, char** argv) : QApplication(argc, argv)
     } else {
         settings->load();
     }
+
+    QString logOutput = commandLineParser.value("log");
+    if (logOutput != "") {
+        if (logOutput == "stderr") {
+            Log::setLogOutput(Log::LogOutput::Stderr);
+        } else if (logOutput == "stdout") {
+            Log::setLogOutput(Log::LogOutput::Stdout);
+        } else if (logOutput == "window") {
+            Log::setLogOutput(Log::LogOutput::Window);
+        } else {
+            Log::setLogOutput(Log::LogOutput::File, logOutput);
```

Comment:
Better to check that the log filename could be opened right here and print the error message in console if we have the console. This is expected behavior on Linux.